src/safe-upgrade-tool/introduction.md

```diff
@@ -0,0 +1,19 @@
+---
+group: software-update-guide
+title: Introduction
+functional_areas:
+  - Upgrade
+---
+
+The Safe Upgrade Tool ALPHA (SUT) is a command line (CLI) tool that checks a Magento instance against a specific version by analyzing all the non-Magento modules installed in it.
+
+The SUT identifies potential problems that must be fixed in your custom code before attempting to upgrade to a newer version of Magento. SUT complements another tool called [Ampersand]((https://github.com/AmpersandHQ/ampersand-magento2-upgrade-patch-helper).
```

Comment:
["Ampersand"](https://ampersandcommerce.com/) is the company behind this tool, the tool itself is called "ampersand-magento2-upgrade-patch-helper". So you'd best change that. Also it looks like there is a `(` too much here.

Reply:
Thanks for the correction @hostep !, much appreciated